packages/orbit-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -188,127 +151,133 @@ export const WithImageInside = ({ size, placement }) => {
+  argTypes: {
+    size: {
+      options: Object.values(SIZE_OPTIONS),
+      control: {
+        type: "select",
+      },
     },
-  },
-  placement: {
-    options: Object.values(PLACEMENTS),
-    control: {
-      type: "select",
+    placement: {
+      options: Object.values(PLACEMENTS),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const Playground = ({
-  content,
-  dataTest,
-  icon,
-  size,
-  tabIndex,
-  enabled,
-  removeUnderlinedText,
-  placement,
-}) => {
-  const Icon = getIcon(icon);
-
-  return (
-    <Tooltip
-      placement={placement}
-      size={size}
-      content={content}
-      dataTest={dataTest}
-      tabIndex={tabIndex}
-      enabled={enabled}
-      removeUnderlinedText={removeUnderlinedText}
-    >
-      <Icon />
-    </Tooltip>
-  );
-};
+export const Playground: Story = {
+  render: ({ children, content, ...args }) => (
+    <Alert icon title="Lorem ipsum dolor sit amet" type="success">
+      <Stack spacing="none">
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Nullam lectus justo, vulputate
+        eget mollis sed, tempor sed magna. Cras elementum. Aliquam erat volutpat. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+        laborum. Sed ac dolor sit amet purus malesuada congue. Sed vel lectus.{" "}
+        <Tooltip
+          content={
+            <div>
+              <div>{content}</div>
+              <TextLink>Clickable element.</TextLink>
+            </div>
+          }
+          {...args}
+        >
+          <Text>{children}</Text>
+        </Tooltip>
+      </Stack>
+    </Alert>
+  ),
 
-Playground.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
-  },
-};
-
-Playground.args = {
-  content: "Write your text here.",
-  dataTest: "test",
-  icon: "Airplane",
-  size: SIZE_OPTIONS.SMALL,
-  tabIndex: "0",
-  enabled: true,
-  removeUnderlinedText: false,
-  placement: PLACEMENTS.BOTTOM,
-};
-
-Playground.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+    info: "You can try all possible configurations of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onShow"],
     },
   },
-  size: {
-    options: Object.values(SIZE_OPTIONS),
-    control: {
-      type: "select",
-    },
+
+  args: {
+    children: "Aliquam erat volutpat.",
+    enabled: true,
+    tabIndex: "0",
+    content:
+      "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ac dolor sit amet purus malesuada congue. Sed vel lectus.",
+    id: "ID",
+    labelClose: "Close",
+    lockScrolling: false,
+    renderInPortal: true,
+    stopPropagation: false,
+    removeUnderlinedText: false,
+    block: false,
+    noFlip: false,
```

Comment:
I looks that `noFlip` and `offset` props are not used in `Tooltip` component at all.

Reply:
I got carried away 😅